datacenter/ucp/2.1/guides/admin/configure/add-labels-to-cluster-nodes.md

```diff
@@ -37,6 +34,33 @@ You can also do this from the CLI by running:
 docker node update --label-add <key>=<value> <node-id>
 ```
 
+## Add constraint to a service
+
+After adding labels to your nodes you can add deployment constraints on your 
+service base on your labels.
+When deploying a service, you can specify constraints, so that the service only
+gets scheduled on a node that has a label that fulfills all the constraints
+you specify
+
+In this example, when users deploy a service they can add constraints for the
+service to be scheduled on nodes that have an SSD storage.
+
+You can apply deployment constraints inside the docker-stack.yml or when you are creating a stack
+![](../../images/use-contraints-in-stack-deployement.png)
+
+Or you can apply them when you are creating a service.
+
+![](../../images/add-constraint-to service-step1.png)
+![](../../images/add-constraint-to service-step2.png)
```

Comment:
A best practice is to add a bit of text between screenshots to guide users. But in this case I think you can just remove the `service-step-1.png` screenshot #ShortDocsFTW